setup.cfg

```diff
@@ -40,6 +40,7 @@ rst-directives = py:data
 [tool:pytest]
 # --cov-report is intentionally empty else pytest-cov will default to generating a report
 addopts =
+  --ddtrace
```

Comment:
@brettlangdon @P403n1x87 Should we do some trick to have this line only in the CI? Something like a separate setup.cfg for CI than for local development?

Reply:
OK, fixed by changing the command in the CI template file